src/ui/pages/group/dialogs/ManageGroupDialog.js

```diff
@@ -84,76 +80,83 @@ const ManageGroupDialog = ({ ...props }) => {
     }
 
     return (
-        <div>
-            <BaseDialog
-                name="Manage Group"
-                IconComponent={InfoOutlinedIcon}
-                actionButtons={
-                    isAdmin ? { "Delete Group": handleDelete, Update: handleUpdate } : { "Leave Group": handleLeave, Update: handleUpdate }
-                }
-                {...props}
-            >
-                <Box sx={{ width: 400 }}>
-                    <Stack spacing={2}>
-                        <MainCard title="Group ID" contentProps={{ component: Stack, spacing: 2, alignItems: "center" }}>
```

Comment:
Changed these to SubCards, and added 
```
contentSX={{ p: 1.5 }}
```
to get a padding value that falls in place nicely